views/js/component/eligibilityEditor.js

```diff
@@ -254,7 +255,20 @@ define([
                     deliveryTreeId : deliveryTreeId,
                     deliveryName : deliveryName
                 })
-                .render($container);
+                .render($container)
+                .resize();
+            },
+
+            /**
+             * Changes eligibilityEditor modal height and adds scroll for the modal to fit the screen height
+             *
+             * @returns {eligibilityEditor} chains the component
+             */
+            resize: function resize() {
+                eligibilityEditor.getElement().css('top', '40px');
+                eligibilityEditor.getElement().css('max-height', ($(window).height() - 80) + 'px');
```

Comment:
Where do those values come from? If I understand well 40 is the top margin, and 80 is then the sum of the top and bottom margins. But why those values, and why is it hardcoded?

Reply:
Okay. It's simple: we cannot guarantee that inital top margin (which is 40px) isn't changed and calculated into another value till this moment. Then, if we are moving all those 'setters' to single call - the actual top margin which will be suitable won't be accessible.
Possibly, this variant would be a little better:
```
eligibilityEditor.getElement().css({
    'top': '40px',
    'max-height': ($(window).height() - 80) + 'px',
    'overflow-y': 'scroll'
});
eligibilityEditor.getElement().css('max-height', ($(window).height() - (parseInt(eligibilityEditor.getElement().css('top'),10)*2)) + 'px');
```

Reply:
Well ok, the values should be fine if you add a comment to explain why.